_pages/en/it-picture-medium-term.md

```diff
@@ -6,11 +6,34 @@ In the following sections, we're trying to take the same end state and create "v
 
 ## Citizen
 
+- Mix of old and new..
+- Current portals and accounts
+- OneGC
+- API Store
+- Pan-Canadian Trust Framework
+- Some benefits integrated with 3rd parties
+
```

Comment:
With the aim that digital services would become an integral part of human lives, greater trust may need to be enforced in services delivered by government through their own portals or 3rd party portals. Should be depict in citizen block somewhere? And how we plan to do it?
Also, just wondering if the portrayal is on how ESDC is perceived by each stakeholder or what benefits ESDC is delivering. I guess, only the language changes here but the context remains the same.

Reply:
From the meeting, I discerned *tangible, tangible, tangible*

So it should be how the citizens perspective changed. "Services are available through central portal" rather than "OneGC implemented" as citizens shouldn't have to care that there is some policy called OneGc, they should only notice their services got easier to use.